zapcli/zap_helper.py

```diff
@@ -330,3 +330,19 @@ def load_session(self, file_path):
             raise ZAPError('No file found at "{0}", cannot load session.'.format(file_path))
         self.logger.debug('Loading session from "{0}"'.format(file_path))
         self.zap.core.load_session(file_path, apikey=self.api_key)
+
+    def xmlreport(self, file_path):
```

Comment:
As a convention, even though this matches the API client method name, we're separating words in method names with underscores, so these two could probably be `xml_report` and `html_report`.